index.html

```diff
@@ -4056,15 +4045,6 @@ <h3>Securing Mechanism Specifications</h3>
 A media type as defined in [[RFC6838]].
           </dd>
           <dt>[=string=] |controller|</dt>
-          <dd>
-A verification method controller as defined in [[VC-DATA-INTEGRITY]] or
-[[VC-JOSE-COSE]].
-          </dd>
-          <dt>[=map=] |controllerDocument|</dt>
-          <dd>
-A controller document as defined in [[VC-DATA-INTEGRITY]] or
-[[VC-JOSE-COSE]].
-          </dd>
```

Comment:
The `controller` DT needs to be deleted here as well.

As a note to the VCWG, @selfissued checked with @awoie (who is the one that raised the issue that resulted in this text) to see if he was ok with removing this text, and he was.

Reply:
`controller` now also deleted